packages/middleware-bucket-endpoint/src/configurations.ts

```diff
@@ -0,0 +1,39 @@
+export interface BucketEndpointInputConfig {
+  bucketEndpoint?: boolean;
+  forcePathStyle?: boolean;
+  preformedBucketEndpoint?: boolean;
+  useAccelerateEndpoint?: boolean;
+  useDualstackEndpoint?: boolean;
+}
```

Comment:
Can you add document block for each of these configs? They should documented be described in API reference